readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -69,13 +70,19 @@ def test_proper_url_no_slash(self):
             },
         ]
 
-        added_versions, deleted_versions = sync_versions_task(
+        self.assertEqual(
+            set(self.pip.versions.all().values_list('slug', flat=True)),
+            {'master', 'latest', 'stable', '0.8.1', '0.8', 'to_delete'},
+        )
+        sync_versions_task(
             self.pip.pk,
             branches_data=branches_data,
             tags_data=[],
         )
-        self.assertEqual(deleted_versions, ['to_delete'])
-        self.assertEqual(added_versions, ['to_add'])
+        self.assertEqual(
+            set(self.pip.versions.all().values_list('slug', flat=True)),
+            {'master', 'latest', 'stable', '0.8.1', '0.8', 'to_add'},
+        )
```

Comment:
the other tests were already checking the versions from the db instead of the return value :)